services/federator/test/unit/Test/Federator/Response.hs

```diff
@@ -58,7 +62,12 @@ testValidationError =
 testServerError :: TestTree
 testServerError =
   testCase "server errors should be converted to wai error responses" $ do
-    resp <- runM . TinyLog.discardLogs . runWaiError @ServerError $ throw InvalidRoute
+    resp <-
+      runM
+        . Log.discardTinyLogs
+        . runWaiError @ValidationError
```

Comment:
This `@ValidationError` is new. Intentional?

Reply:
Thanks for spotting! This was not intentional.

Reply:
Removed in b4b3ae05a.